collector/writer.go

```diff
@@ -29,56 +32,108 @@ func (w CollectdWriter) Write(s Stats) error {
 }
 
 func (w CollectdWriter) writeInts(s Stats) error {
-	metrics := map[string]uint64{
-		"cpu.user":   s.Stats.CPUStats.CPUUsage.UsageInUsermode,
-		"cpu.system": s.Stats.CPUStats.CPUUsage.UsageInKernelmode,
-		"cpu.total":  s.Stats.CPUStats.CPUUsage.TotalUsage,
-
-		"memory.limit": s.Stats.MemoryStats.Limit,
-		"memory.max":   s.Stats.MemoryStats.MaxUsage,
-		"memory.usage": s.Stats.MemoryStats.Usage,
-
-		"memory.active_anon":   s.Stats.MemoryStats.Stats.TotalActiveAnon,
-		"memory.active_file":   s.Stats.MemoryStats.Stats.TotalActiveFile,
-		"memory.cache":         s.Stats.MemoryStats.Stats.TotalCache,
-		"memory.inactive_anon": s.Stats.MemoryStats.Stats.TotalInactiveAnon,
-		"memory.inactive_file": s.Stats.MemoryStats.Stats.TotalInactiveFile,
-		"memory.mapped_file":   s.Stats.MemoryStats.Stats.TotalMappedFile,
-		"memory.pg_fault":      s.Stats.MemoryStats.Stats.TotalPgfault,
-		"memory.pg_in":         s.Stats.MemoryStats.Stats.TotalPgpgin,
-		"memory.pg_out":        s.Stats.MemoryStats.Stats.TotalPgpgout,
-		"memory.rss":           s.Stats.MemoryStats.Stats.TotalRss,
-		"memory.rss_huge":      s.Stats.MemoryStats.Stats.TotalRssHuge,
-		"memory.unevictable":   s.Stats.MemoryStats.Stats.TotalUnevictable,
-		"memory.writeback":     s.Stats.MemoryStats.Stats.TotalWriteback,
+	totalPercent, userPercent, systemPercent := calculateCPUPercent(s.Stats.PreCPUStats, s.Stats.CPUStats)
+	cpu_metrics := map[string]float64{
+		"vcpu-user":   userPercent,
+		"vcpu-system": systemPercent,
+		"vcpu-total":  totalPercent,
 	}
 
+	memory_metrics := map[string]uint64{
+		"memory-limit": s.Stats.MemoryStats.Limit,
```

Comment:
The standard format is ```host "/" plugin ["-" plugin instance] "/" type ["-" type instance]```     If you have EscapeCharacter set to _ this ends up changing memory.limit to memory_limit.  We could probably make that configurable in the app if you feel strongly about it.